src/modules/dashboard/components/ActionsPage/MultisigWidget/MultisigWidget.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { ReactNode, useState } from 'react';
+import { ColonyRole } from '@colony/colony-js';
+import { nanoid } from 'nanoid';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { Address } from '~types/index';
+import styles from './MultisigWidget.css';
```

Comment:
`styles` should be the last import.

Also please separate (w/ a new line), external module imports (coming from `node_modules`) from our local modules imports (eg: `dashboard`, `core`)_

Thanks